packages/backend/src/services/action.ts

```diff
@@ -71,6 +71,10 @@ export const processAction = async (options: ProcessActionOptions) => {
     }
   } catch (error) {
     logger.error(error)
+    // log raw http error from StepError
+    if (error.cause) {
+      logger.error(error.cause)
+    }
     if (error instanceof HttpError) {
```

Comment:
keep status quo and update these references appropriately. We don't want to lose data until we make a decision on whether we want to stop logging raw error details inside `actionOutput`.

example: 
```
   // TODO: make decision on StepError logging strategy
    const underlyingError = error instanceof StepError ? error.cause : error

    if (underlyingError instanceof HttpError) {
      $.actionOutput.error = {
        details: underlyingError.details,
        status: underlyingError.response.status,
        statusText: underlyingError.response.statusText,
      }
    } else {
      try {
        $.actionOutput.error = JSON.parse(underlyingError.message)
      } catch {
        $.actionOutput.error = { error: underlyingError.message }
      }
    }
```

Reply:
Discussed offline; this borks `executeFlow`. Going ahead with alt method: adding `details` to IStepError or similar.